ledger/src/blockstore/blockstore_purge.rs

```diff
@@ -444,15 +441,18 @@ impl Blockstore {
     /// deserializing each slot being purged and iterating through all
     /// transactions to determine the keys of individual records.
     ///
+    /// The purge range applies to \[`from_slot`, `to_slot`\].
+    ///
     /// **This method is very slow.**
     fn purge_special_columns_exact(
         &self,
         batch: &mut WriteBatch,
         from_slot: Slot,
-        to_slot: Slot, // Exclusive
+        to_slot: Slot,
     ) -> Result<()> {
         let mut index0 = self.transaction_status_index_cf.get(0)?.unwrap_or_default();
         let mut index1 = self.transaction_status_index_cf.get(1)?.unwrap_or_default();
+        let to_slot = to_slot.saturating_add(1);
```

Comment:
nit: there's more rusty way: 

```rust
for slot in from_slot..=to_slot {
    ...
}
```

Reply:
Thanks for spotting this.  Will have a quick fix for this.